doc/sphinxman/source/dftnl.rst

```diff
@@ -0,0 +1,73 @@
+.. #
+.. # @BEGIN LICENSE
+.. #
+.. # Psi4: an open-source quantum chemistry software package
+.. #
+.. # Copyright (c) 2007-2018 The Psi4 Developers.
+.. #
+.. # The copyrights for code used from other parties are included in
+.. # the corresponding files.
+.. #
+.. # This file is part of Psi4.
+.. #
+.. # Psi4 is free software; you can redistribute it and/or modify
+.. # it under the terms of the GNU Lesser General Public License as published by
+.. # the Free Software Foundation, version 3.
+.. #
+.. # Psi4 is distributed in the hope that it will be useful,
+.. # but WITHOUT ANY WARRANTY; without even the implied warranty of
+.. # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+.. # GNU Lesser General Public License for more details.
+.. #
+.. # You should have received a copy of the GNU Lesser General Public License along
+.. # with Psi4; if not, write to the Free Software Foundation, Inc.,
+.. # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+.. #
+.. # @END LICENSE
+.. #
+
+.. include:: autodoc_abbr_options_c.rst
+
+.. index:: DFTNL
+.. _`sec:dftnl`:
+
+
+DFT-NL
+======
+
+.. sectionauthor:: Holger Kruse
+
+Non-local (NL), density based correlation energy from the VV10 kernel can be added
+to arbitrary functionals.
+
+.. math:: E_{DFT-NL}=E_{DFT}+E_{NL}
+
+For pre-defined functionals (see Functional overview in :ref:`this Table <table:dft_all>` ) it is sufficient to add `-NL` to 
+the functional name::
+
+    energy('b3lyp-nl')
+
+Modification of the parameters `b` and `C` is done setting |scf__DFT_VV10_B| and |scf__DFT_VV10_C|. The `C` is usually left unchanged and the originally proposed
+value of `C=0.0093` is used. 
+
+Adding |scf__DFT_VV10_B| to any functional activates the calculation of the VV10 kernel. A BLYP-NL calculation can be set as follows::
+   
+    set DFT_VV10_B 4.0
+    energy('blyp')
+
+The default `C` parameter will be used.
+
+Similar to |scf_DFT_DISPERSION_PARAMETERS| the tuple |scf_NL_DISPERSION_PARAMETERS| can used::
+    
+    set NL_DISPERSION_PARAMTERS [4.0]
+    energy('blyp')
```

Comment:
Someday I want to make sure that an `energy('b3lyp')` doesn't produce a `B3LYP TOTAL ENERGY` that is anything but the certified B3LYP energy. But that involved checking alpha, -D, -NL, etc. so way out of the frame of this PR.